lib/src/front_end/comment_writer.dart

```diff
@@ -263,7 +263,7 @@ class CommentSequence extends ListBase<SourceComment> {
     // Doc comments and non-inline `/* ... */` comments are always pushed to
```

Comment:
Nit: It would be nice if the comment for this was also updated with the condition changed. It seems to be like the complement of the condition now.

Reply:
Documented both.